src/Loading/sceneLoader.ts

```diff
@@ -976,4 +978,143 @@ export class SceneLoader {
             }, pluginExtension);
         });
     }
+
+    /**
+     * Import animations from a file into a scene
+     * @param rootUrl a string that defines the root url for the scene and resources or the concatenation of rootURL and filename (e.g. http://example.com/test.glb)
+     * @param sceneFilename a string that defines the name of the scene file or starts with "data:" following by the stringified version of the scene or a File object (default: empty string)
+     * @param scene is the instance of BABYLON.Scene to append to (default: last created scene)
+     * @param targetConverter defines a function used to convert animation targets from loaded scene to current scene (default: search node by name)
+     * @param onSuccess a callback with the scene when import succeeds
+     * @param onProgress a callback with a progress event for each file being loaded
+     * @param onError a callback with the scene, a message, and possibly an exception when import fails
+     */
+    public static ImportAnimations(rootUrl: string, sceneFilename: string | File = "", scene: Nullable<Scene> = EngineStore.LastCreatedScene, targetConverter: Nullable<(target: any) => any> = null, onSuccess: Nullable<(scene: Scene) => void> = null, onProgress: Nullable<(event: SceneLoaderProgressEvent) => void> = null, onError: Nullable<(scene: Scene, message: string, exception?: any) => void> = null): void {
+        if (!scene) {
+            Logger.Error("No scene available to load animations to");
+            return;
+        }
+
+        let _targetConverter = targetConverter ? targetConverter : this._defaultTargetConverter(scene);
+
+        let onAssetContainerLoaded = (container: AssetContainer) => {
+            SceneLoader.MergeAnimations(scene, container, _targetConverter);
+
+            container.dispose();
+
+            if (onSuccess) {
+                onSuccess(scene);

```

Comment:
Maybe we could add a global observable on the scene,  something like: scene.onAnimationMergedObservable so the sandbox can listen to it and update accordingly

Reply:
Ok!

Reply:
Done in https://github.com/BabylonJS/Babylon.js/pull/7252/commits/3130b945ece58712d89339e861b6db08af9eb5e3